.gitignore

```diff
@@ -0,0 +1,9 @@
+PythonTool/visitlog.py
+Data/*.vtk
+Data/*.stl
+Data/*.h5m
+Data/meshtal
+Sessions
+Images
+PythonTool/*.pyc
+BashTool/DataFileInfo/*.txt
```

Comment:
Sessions: To ignore generated XML sessions when pushing.
Images: To ignore generated images when pushing.
PythonTool/*.pyc: Each time the tool is run, *.pyc files are generated that are not needed for pushing into a repository.
BashTool/DataFileInfo/*.txt: I removed it after modifying bash scripts to save to variables instead of text files.